src/index.html

```diff
@@ -9,11 +9,12 @@
   <link rel="icon" type="image/png" sizes="32x32" href="images/favicon-32x32.png">
   <link rel="icon" type="image/png" sizes="16x16" href="images/favicon-16x16.png">
   <link rel="manifest" href="assets/site.webmanifest">
-  <link rel="preload" href="titilliumweb-regular-webfont.woff2" as="font" type="font/woff2" crossorigin="anonymous">
-  <link rel="preload" href="titilliumweb-bold-webfont.woff2" as="font" type="font/woff2" crossorigin="anonymous">
-  <link rel="preload" href="titilliumweb-light-webfont.woff2" as="font" type="font/woff2" crossorigin="anonymous">
-  <link rel="preload" href="titilliumweb-italic-webfont.woff2" as="font" type="font/woff2" crossorigin="anonymous">
-  <link rel="preload" href="primeicons.ttf" as="font" type="font/ttf" crossorigin="anonymous">
```

Comment:
I didn't see the loading of prime icons on landing to app, before it was https://dev.algorea.org/branch/feature/do-not-block-ui-on-font-load-postbuild/en/activities/by-id/4702;path=;parentAttempId=0

So for now I've removed it